package/tests/integration/playwright.config.ts

```diff
@@ -1,16 +1,16 @@
-import { defineConfig, devices } from '@playwright/test';
+import { defineConfig } from '@playwright/test';
 
 export default defineConfig({
     outputDir: './.test-results',
     preserveOutput: 'failures-only',
     forbidOnly: !!process.env.CI,
-    retries: process.env.CI ? 2 : 0,
     workers: process.env.CI ? 1 : undefined,
-    timeout: 5_000,
-    expect: { timeout: 5_000 },
-    reporter: 'line',
```

Comment:
Line is the default outside of CI, so configuring it is not necessary

Reply:
Thank you